postcss.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  plugins: [require("postcss-bidirection"), require("autoprefixer")]
```

Comment:
I bet we can add postcss.config.js to the list of exported files and avoid this:

https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-launchpad/package.json#L113